src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -786,7 +759,8 @@ private void ShowAddFileDialogAndAdd()
                 fDialog = new OpenFileDialog()
                 {
                     Filter = "Custom Node, DLL, XML (*.dyf, *.dll, *.xml)|*.dyf;*.dll;*.xml",
-                    Title = "Add Custom Node, Library, or XML file to Package..."
+                    Title = "Add Custom Node, Library, or XML file to Package...",
+                    Multiselect = true
                 };
```

Comment:
Strangely I thought I added this a long time ago but it has since regressed... Oh well!